File: backend/voting-app/src/main/java/com/tu/votingapp/dto/response/UserDTO.java
package com.tu.votingapp.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDTO {
    private Long id;
    private String name;
    private String email;
    private String phone;
    private String currentAddress;
    private Long locationId;
    private String egn;
    private DocumentDTO document;
    private List<RoleDTO> roles;
}

File: frontend/android/app/src/main/java/com/example/android/dto/response/UserDTO.kt
package com.example.android.dto.response

import kotlinx.serialization.Serializable
import kotlinx.serialization.Transient

@Serializable
data class UserDTO(
    val id: Long? = null,
    val name: String?,
    val email: String?,
    val phone: String?,
    val password: String?,
    val currentAddress: String?,
    val locationId: LocationResponseDTO?,
    val egn: String?,
    val document: DocumentDTO? = null,
    @Transient val roles: RoleDTO? = null
)